test/gs.uefa.test.js

```diff
@@ -8,12 +8,19 @@ test('uefa', function *(t) {
   t.eq(gs.findMatches({s: 1}).length, 3, '3 matches per group');
 
   opts.meetTwice = true;
+  opts.numberOfDuels = 3;
+  var uefa = new GS(6, opts);
+  t.eq(uefa.findMatches({s: 1}).length, 9, '9 matches per group');
```

Comment:
you should test that every player is in 3x2 matches per group i think

Reply:
Ok!